app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminOutOfStockThresholdOnConfigurationPageTest.xml

```diff
@@ -10,86 +10,74 @@
-            <magentoCLI command="cache:flush" stepKey="cleanCacheAfterDisableManageStock1"/>
+            <createData entity="SimpleSubCategory" stepKey="category"/>
+            <actionGroup ref="LoginAsAdmin" stepKey="loginAsAdmin"/>
+            <createData entity="SimpleProduct" stepKey="simpleProduct">
+                <requiredEntity createDataKey="category"/>
+            </createData>
+            <amOnPage url="{{AdminProductEditPage.url($$simpleProduct.id$$)}}" stepKey="openProductEditPageToAssignSource"/>
+            <actionGroup ref="AssignSourceToProductActionGroup" stepKey="assignUSSource1ToCreatedProduct">
+                <argument name="sourceCode" value="$$customSource.source[source_code]$$"/>
+            </actionGroup>
+            <fillField selector="{{AdminProductSourcesGrid.rowQty('1')}}" userInput="100" stepKey="setCustomSourceQuantity"/>
+            <actionGroup ref="UnassignSourceFromProductActionGroup" stepKey="unassignSource">
+                <argument name="sourceCode" value="{{_defaultSource.name}}"/>
+            </actionGroup>
+            <click selector="{{AdminProductFormActionSection.saveButton}}" stepKey="saveEditedSimpleProduct"/>
         </before>
+
         <after>
-            <!--Assign Default Stock to Default Website.-->
             <actionGroup ref="AssignWebsiteToStockActionGroup" stepKey="assignMainWebsiteToDefaultStock">
                 <argument name="stockName" value="{{_defaultStock.name}}"/>
                 <argument name="websiteName" value="{{_defaultWebsite.name}}"/>
             </actionGroup>
-            <magentoCLI command="config:set cataloginventory/item_options/min_qty 1" stepKey="setNotifyForQuantityBelowToOne"/>
-            <magentoCLI command="indexer:reindex" stepKey="performReindexAfterDisableManageStock1"/>
-            <magentoCLI command="cache:flush" stepKey="cleanCacheAfterDisableManageStock1"/>
-
-            <actionGroup ref="logout" stepKey="logoutOfAdmin1"/>
+            <actionGroup ref="DisableSourceActionGroup" stepKey="disableCreatedCustomSource1">
+                <argument name="sourceCode" value="$$customSource.source[source_code]$$"/>
+            </actionGroup>
+            <deleteData createDataKey="category" stepKey="deleteCategory"/>
+            <deleteData createDataKey="CustomStockMainWebsite" stepKey="deleteStock"/>
+            <deleteData createDataKey="simpleProduct" stepKey="deleteProduct"/>
+            <!-- Revert out of stock threshold value in config. -->
+            <magentoCLI command="config:set cataloginventory/item_options/min_qty 0" stepKey="setMinQtyToDefaultValue"/>
+            <magentoCLI command="indexer:reindex" stepKey="runIndexerReindex" />
+            <magentoCLI command="cache:flush" stepKey="cleanCache"/>
+            <waitForPageLoad time="30" stepKey="waitForIndexAndCache"/>
+            <actionGroup ref="logout" stepKey="logoutOfAdmin"/>
```

Comment:
Move delete action and cli commands in first lines on after, as mentioned in previous comment